web/modules/custom/mymodule/config/install/mymodule.settings.yml

```diff
@@ -0,0 +1,5 @@
+enable: false
```

Comment:
@anitha-innoraft - What is this setting for?

Reply:
This setting with boolen value is for make it editable.